db/migrate/20180720152009_create_phone_configurations_table.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePhoneConfigurationsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :phone_configurations do |t|
+      t.references :user, null: false
+      t.string :phone_fingerprint, default: '', null: false
+      t.text :encrypted_phone, null: false
+      t.integer :delivery_preference, default: 0, null: false
+      t.timestamp :confirmation_sent_at
+      t.timestamp :confirmed_at
+      t.timestamps
```

Comment:
We also need a boolean field to track whether or not this phone is to be used for 2FA. Maybe something like `can_receive_otp`.

Reply:
-nod- -- I'm working on a commit that cleans this up a little and adds an `enabled` flag that's `true` by default.

Reply:
If our goal is to find a name that could also apply to other 2FA methods in case we end up allowing more than one, then how about something like `use_for_2fa` or `used_for_2fa`, or `two_factor_enabled`?

Reply:
I'm only trying to replicate the uses we have for the user phone setting, not the LOA3 stuff.